test/Numerics/DGMethods/compressible_navier_stokes_equations/three_dimensional/test_buoyancy.jl

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env julia --project
+
+include("../boilerplate.jl")
+include("ThreeDimensionalCompressibleNavierStokesEquations.jl")
+
+ClimateMachine.init()
+
+#################
+# RUN THE TESTS #
+#################
+@testset "$(@__FILE__)" begin
+
+    include("refvals_buoyancy.jl")
+
+    ########
+    # Setup physical and numerical domains
+    ########
+    Ωˣ = IntervalDomain(-2π, 2π, periodic = true)
+    Ωʸ = IntervalDomain(-2π, 2π, periodic = true)
+    Ωᶻ = IntervalDomain(0, 4π, periodic = false)
+
+    second_order = DiscretizedDomain(
+        Ωˣ × Ωʸ × Ωᶻ;
+        elements = 5,
+        polynomial_order = 2,
+        overintegration_order = 1,
+    )
+
+    fourth_order = DiscretizedDomain(
+        Ωˣ × Ωʸ × Ωᶻ;
+        elements = 3,
+        polynomial_order = 4,
+        overintegration_order = 1,
+    )
+
+    grids = Dict("second_order" => second_order, "fourth_order" => fourth_order)
+
+    ########
+    # Define timestepping parameters
+    ########
+    start_time = 0
+    end_time = 0.1
+    Δt = 0.001
+    method = SSPRK22Heuns
+
+    timestepper = TimeStepper(method = method, timestep = Δt)
+
+    callbacks = (
+        Info(),
+        # VTKState(; iteration = 1, filepath = "output/buoyancy"),
+        StateCheck(10),
+    )
+
+    ########
+    # Define physical parameters and parameterizations
+    ########
+    Lˣ, Lʸ, Lᶻ = length(Ωˣ × Ωʸ × Ωᶻ)
+    parameters = (
+        ρₒ = 1, # reference density
+        cₛ = sqrt(10), # sound speed
+        α = 1e-4, # thermal expansion coefficient
+        g = 10, # gravity
+        θₒ = 10, # initial temperature value
+        Lˣ = Lˣ,
+        Lʸ = Lʸ,
+        H = Lᶻ,
+    )
+
+    physics = FluidPhysics(;
+        advection = NonLinearAdvectionTerm(),
+        dissipation = ConstantViscosity{Float64}(μ = 0, ν = 0, κ = 0),
+        coriolis = nothing,
+        buoyancy = Buoyancy{Float64}(α = parameters.α, g = parameters.g),
+    )
+
+    ########
+    # Define initial conditions
+    ########
+
+    u₀(p, x, y, z) = -0
+    v₀(p, x, y, z) = -0
+    w₀(p, x, y, z) = -0
+    θ₀(p, x, y, z) = -p.θₒ * (1 - z / 4π)
+
+    ρ₀(p, x, y, z) =
+        p.ρₒ * (1 - (p.α * p.g / p.cₛ^2) / 2 * (-p.θₒ * (1 - z / 4π))^2)
+    ρu₀(p, x...) = ρ₀(p, x...) * u₀(p, x...)
+    ρv₀(p, x...) = ρ₀(p, x...) * v₀(p, x...)
+    ρw₀(p, x...) = ρ₀(p, x...) * w₀(p, x...)
+    ρθ₀(p, x...) = ρ₀(p, x...) * θ₀(p, x...)
+
+    ρu⃗₀(p, x...) = @SVector [ρu₀(p, x...), ρv₀(p, x...), ρw₀(p, x...)]
+    initial_conditions = (ρ = ρ₀, ρu = ρu⃗₀, ρθ = ρθ₀)
```

Comment:
More unrecognizable characters. Is this just me? cc @kpamnany 

<img width="578" alt="Screen Shot 2021-03-16 at 2 40 53 PM" src="https://user-images.githubusercontent.com/1880641/111383579-a734b400-8665-11eb-8770-03ad516491ec.png">

Reply:
https://juliamono.netlify.app/ exists and it's not that much harder to install than julia itself

Reply:
They are blank blocks for me as well. I don't think it's wise to require that someone install a font just to read our code.

Reply:
Works fine for me :).

Reply:
Blank blocks for me.

Reply:
https://github.com/CliMA/ClimateMachine.jl/blob/5165fc041efa68c02fd87f2fb0a3492ca2d05d06/src/Common/Orientations/Orientations.jl#L89
https://github.com/CliMA/ClimateMachine.jl/blob/5165fc041efa68c02fd87f2fb0a3492ca2d05d06/src/Atmos/Model/AtmosModel.jl#L461

I mean I can do this 
![Screen Shot 2021-03-17 at 12 36 17](https://user-images.githubusercontent.com/7977055/111504487-179f0c00-871e-11eb-96f3-477bec9c5f27.png)
but I think that is a decrease in code readability

Reply:
Without the font, it's an even greater decrease in readability. See: https://clima.github.io/ClimateMachine.jl/latest/DevDocs/AcceptableUnicode/

Reply:
I think the acceptable unicode page needs an update :). We need to be able to distinguish vectors from scalars.

Reply:
I admit, it is helpful to have `\hat` sometimes.

Reply:
> Uh, okay. So let's just ignore them then.

Not, not ignore.

Reply:
@kpamnany @charleskawczynski what browser are you using? I see it fine in Firefox.

Reply:
Chrome

Reply:
Brave

Reply:
Firefox apparently uses Menlo Regular for this item which renders it fine (also fine if I paste it into my terminal and use Menlo Regular). Can you check want font your browser is using? 

(I did it by right click on the text, click inspect element, then there was a "Fonts" tab which tells you what font it is)

Reply:
Interesting, Brave on my phone renders fine

Reply:
Can you check what font it is using?

Reply:
`font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;`

Reply:
Brave on desktop: `font-family: SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace;`. Not sure how to check it on the phone.

Reply:
That's the font list. Is there a way to check which font is actually used?